imagine-dragons-web/components/nav.html

```diff
@@ -1,10 +1,21 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
 <header>
     <nav>
         <ul>
-            <li><a href="home.html">Home Page</a></li>
-            <li><a href="songs.html">Songs</a></li>
-            <li><a href="tours.html">Tours</a></li>
-            <li><a href="contact.html">Contact</a></li>
+            <li><i class="fa fa-home" aria-hidden="true"></i> <a href="home.html">Home Page</a></li>
```

Comment:
what is `aria-hidden="true"`? why is it relevant?

Reply:
All aria properties were invented for people with disabilities - meaning that if a blind person use the computer, he would probably use a screen-reader (due to the fact he can not see the screen by himself), the use of aria-hidden make the screen reader to ignore the content that has this property.
Now, because blind people won't use my computer, this property is reluctant.